events/wc_frostwolf_events.txt

```diff
@@ -88,11 +127,11 @@ province_event = {
 		d_frostwolf = {
 			holder_scope = {
 				if = {
-					limit = { frostwolf_exodus_d_frostwolf_trigger = yes }
+					limit = { frostwolf_leader_exists = yes }
 					narrative_event = { id = WCFW.10 }
 				}
 				else = {
-					character_event = { id = WCFW.200 }
+					73 = { province_event = { id = WCFW.160 } }
```

Comment:
There's big problem with this event. `WCFW.200` was fail notification to the Frostwolf leader. If the Frostwolf leader fails migration now, he will send event to the Stormreaver leader... but if there's no Stormreaver leader... you get no event nor notifaction. The event chain just stops without informing what happend.

Reply:
I'll need to think about how I'll handle this.. Perhaps at that point the `e_horde` leader takes care of it and sees them running off as betrayal? Considering it all started as a secret deal between Stormreaver and Frostwolves it would stand to reason he sees it as betrayal.

I'll write a separate chain of events for that if you think it should go them. If not, feel free to suggest. I'll ponder over this a bit as I'm eating dinner.. Good point nonetheless.

Reply:
I suggest returning `WCFW.200` notification. The Frostwolves always need it, they should know that their migration failed when it's failed.

Reply:
I'll write a separate failure event that plays for the frostwolves only then that calls itself regardless if a Stormreaver lives. How's that sound?

Reply:
That sounds like returning of `WCFW.200`, so I'm good.